x-pack/plugins/security_solution/server/endpoint/routes/policy/index.ts

```diff
@@ -37,4 +42,13 @@ export function registerPolicyRoutes(router: IRouter, endpointAppContext: Endpoi
     },
     getAgentPolicySummaryHandler(endpointAppContext)
   );
+
+  router.get(
+    {
+      path: BASE_POLICY_ROUTE,
+      validate: false,
```

Comment:
need to add the schema from fleet

Reply:
Can you include the schema validation prior to committing this?
The schema you want is `GetPackagePoliciesSchema` located here:
https://github.com/elastic/kibana/blob/b1253db197c2af1eb88458b39dfc0322315db529/x-pack/plugins/fleet/server/types/rest_spec/package_policy.ts#L14-L16

I'm not srue if you can import directly from `fleet/server/types`. if not, then just export it out of fleet's `server/index`